### PR TITLE
ARXIVNG-3675: licenses

### PR DESCRIPTION
**Pull request by mhl10**
This updates the license page per the google doc.

Screen Shot 2020-11-02 at 8 41 01 AM
Screen Shot 2020-11-02 at 8 41 08 AM

It's also [staged](https://beta-new.arxiv.org/help/license) on the new beta instance, but it's currently locked down, so I included above screenshots.

### CODE REVIEW

File: help/license.md [hunk 1]
```diff
@@ -1,103 +1,73 @@
...
-granting a publication license. Please check the policies of any
-potential publication venue before uploading to arXiv. (For the policy
-information of many publishers, see the
-[SHERPA/RoMEO](http://www.sherpa.ac.uk/romeo.php) site.)
+----------------
+A [Creative Commons CC0 1.0 Universal Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/) will apply to all metadata.
 
 Copyright notices
 -----------------
+Often, PDFs found on publisher websites or supplied as proofs are the property of the publisher, which often owns the copyright and/or licenses their use. Submission of PDFs with copyright statements which prohibit or impair arXiv's redistribution license will be rejected. If explicit permission is given for open access redistribution, these are permissible.
```
Comment (jimentwood) [inline help/license.md:69]:
```suggestion
Often, PDFs found on publisher websites or supplied as proofs are the property of the publisher, which often owns the copyright and/or licenses their use. Submission of PDFs with copyright statements which prohibit or impair arXiv's redistribution license will be rejected. If explicit permission is given for open access redistribution, these are permissible.
Users should contact arXiv administrators at help@arxiv.org if they have questions.
```
This was added recently as we had out of band communication. Would prefer to keep it for now.

File: help/license.md [hunk 2]
```diff
@@ -1,103 +1,73 @@
...
+-----------------------
+
+arXiv values open access and encourages choosing a liberal license for re-use of the paper. However, each situation is unique. Funders may require specific licenses. Journals have different policies regarding preprints, embargo periods, and post-publication distribution. Please note that different versions of the work can have different licenses.
+
+Ultimately, it is the submitter’s responsibility to choose the appropriate license for their content. The license chosen is irrevocable and cannot be changed.
+
+To make a license decision, arXiv recommends the following:
+1. Read and understand the [submission terms and agreement](policies/submission_agreement) and [instructions for submission](policies/instructions_for_submission)
+2. Understand the meanings and implications of each license below
+3. Check the policies of funders and journals (contact journals directly or using a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html)
```
Comment (jimentwood) [inline help/license.md:31]:
```suggestion
3. Check the policies of funders and journals (contact journals directly or using a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html))

```

Comment (afromme) [reply help/license.md:31]: I agree with this suggestion

**Review by afromme — APPROVED** [review]
approve after incorporating JE's suggestions

**Review by SBBCornell — COMMENTED** [review]
Ive just created an icon for arXiv similar to the existing CC licenses. If this gets a thumbs up I can push a commit to include the vector icon on the page:
Screen Shot 2020-11-02 at 9 07 37 AM

**Comment by afromme** [comment]
Yes to the new arXiv license icon!

**Comment by SBBCornell** [comment]
This seems like a good opportunity to draw on our extended style options for markdown pages in order to differentiate between the licenses (https://github.com/arXiv/arxiv-docs/tree/develop/styles). Here is a quick screenshot with blockquotes in use:
Screen Shot 2020-11-02 at 9 24 40 AM

If this looks good I can push a commit

**Comment by afromme** [comment]
Excellent!

**Review by jimentwood — APPROVED** [review]
looks good - with request to add back in the line about emailing help

File: help/license.md [hunk 3]
```diff
@@ -1,103 +1,85 @@
...
+-----------------------
+
+arXiv values open access and encourages choosing a liberal license for re-use of the paper. However, each situation is unique. Funders may require specific licenses. Journals have different policies regarding preprints, embargo periods, and post-publication distribution. Please note that different versions of the work can have different licenses.
+
+Ultimately, it is the submitter’s responsibility to choose the appropriate license for their content. The license chosen is irrevocable and cannot be changed.
+
+To make a license decision, arXiv recommends the following:
+1. Read and understand the [submission terms and agreement](policies/submission_agreement) and [instructions for submission](policies/instructions_for_submission)
+2. Understand the meanings and implications of each license below
+3. Check the policies of funders and journals (contact journals directly or using a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html))
```
Comment (afromme) [inline help/license.md:41]: Check the policies of funders and journals (contact journals directly or use a third-party service such as [Sherpa Romeo](https://v2.sherpa.ac.uk/romeo/about.html))

**Pull request merged**